public/app/features/trails/shared.ts

```diff
@@ -15,7 +15,8 @@ export const TRAILS_ROUTE = '/explore/metrics/trail';
 export const HOME_ROUTE = '/explore/metrics';
 
 export const VAR_FILTERS = 'filters';
-export const VAR_FILTERS_EXPR = '{${filters}}';
+export const VAR_FILTERS_WITH_CURLY_EXPR = '{${filters}}';
```

Comment:
Haha, I see what you did there. This is only used in the base query where we wrap the variable in `{}`for the query. It would be nice to have a little code comment describing this. It is the only constant variable  in this file that has this special extra set of characters.

Reply:
I decide to remove this and have one variable. and Put the comments where we use them

Reply:
@bohandley Here is the change https://github.com/grafana/grafana/pull/97815/commits/deb1fe008b0384fc3a19a42eb437572f48e98cf4